model/error.yaml

```diff
@@ -0,0 +1,34 @@
+groups:
+  - id: error
+    type: attribute_group
+    prefix: error
+    brief: >
+      This document defines the shared attributes used to
+      report an error.
+    attributes:
+      - id: description
+        brief: 'Describes a class of error operation has ended with.'
+        type:
+          allow_custom_values: true
+          members:
+            - id: empty
+              value: ""
```

Comment:
How is this different from not set?

Reply:
it's not different - some metric systems (e.g. Prometheus) treat lack of value and empty value the same, this reserves empty value to represent no-error case for everyone.